src/common/queries/useCurrentPrices.ts

```diff
@@ -12,14 +12,15 @@ const getHistoricalStxPrice = async ({ queryKey }: QueryFunctionContext) => {
 export const useSuspenseStxPrice = (
   blockBurnTime?: string,
   options?: Partial<UseQueryOptions<any, unknown, any, string[]>>
-) =>
-  useSuspenseQuery({
-    queryKey: ['stx-price', blockBurnTime || 'current'],
+) => {
+  return useSuspenseQuery({
+    queryKey: ['stx-price', blockBurnTime ? blockBurnTime.split('T')[0] : 'current'],
```

Comment:
It feels like this should be a var too because it's not clear what value this splitting is getting out